File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    int block = 1;

    // iterate over each row
    for (int i = 0; i < height; i++) {
        // iterate over each column
        // print spaces
        for (int space = 0; space < height - block; space++) {
            printf(" ");
        }
        // print blocks
        for (int j = 0; j < block; j++) {
            printf("#");
        }
        block++;
        printf("\n");
    }
}

File: pset6/mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

brick = 1

for i in range(height):
    for j in range(height - brick):
        print(" ", end="")
    for k in range(brick):
        print("#", end="")
    print("\n", end="")
    brick += 1